pyjelly/integrations/generic/parse.py

```diff
@@ -312,7 +333,13 @@ def parse_jelly_to_graph(
     options, frames = get_options_and_frames(inp)
     sink = sink_factory()
 
-    for item in parse_jelly_flat(inp=inp, frames=frames, options=options):
+    for item in parse_jelly_flat(
+        inp=inp,
+        frames=frames,
+        options=options,
+        logical_type_strict=False,
+        **{"frame_metadata": frame_metadata} if frame_metadata is not None else {},
```

Comment:
We don't have to support metadata in flat streams, it would be very hard to use anyway. Simplify, just implement it for grouped streams.